src/components/PaymentLog.vue

```diff
@@ -576,12 +581,10 @@ export default {
     compHeader() {
       return this.isAltaraPay ? "Altara Pay" : "Altara Credit";
     },
-    
   },
-
   methods: {
     watchSalesLogForm(){
-     this.salesLogForm.discount = this.salesLogForm?.sales_category_id == "2" ? "5_discount" : "0_discount"
+     this.salesLogForm.discount = ((this.salesLogForm?.sales_category_id == "2") && !(this.salesLogForm.product_name.includes("Cash")|| this.salesLogForm.product_name.includes("cash") )) ? "5_discount" : "0_discount"
```

Comment:
convert `this.salesLogForm.product_name` to `lowercase` or `uppercase` before checking to avoid the `or` statement.